tests/test_formatters.py

```
@@ -206,524 +206,526 @@
+
+    # yd_format_from_day()
+    assert yd_format_from_day(0) == '2000-001'
+
+    assert np.all(yd_format_from_day([-365,0,366])
+                           == ['1999-001', '2000-001', '2001-001'])
+
+    # Check if yd_format_from_day start from 2000-001
+    assert yd_format_from_day(0) == '2000-001'
+
+    # Errors
+    with pytest.raises(ValueError):
```

Ruff (0.15.6)

[warning] `pytest.raises(ValueError)` is too broad, set the `match` parameter or use a more specific exception

(PT011)